.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-publish:
```

Comment:
Is this consistent with the naming we've used elsewhere? Couldn't it just be `publish`?

Reply:
Yes, and it will be :smile: